packages/react/src/runtimes/remote-thread-list/RemoteThreadListThreadListRuntimeCore.tsx

```
@@ -0,0 +1,386 @@
+    while (
+      this._state.baseValue.newThreadId !== undefined &&
+      this._state.value.newThreadId === undefined
+    ) {
+      await this._state.waitForUpdate();
+    }
+
+    const state = this._state.value;
+    let threadId: string | undefined = this._state.value.newThreadId;
+    if (threadId === undefined) {
+      do {
+        threadId = "__LOCALID_" + generateId();
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)